src/frontend/src/styles/index.css

```diff
@@ -73,15 +73,6 @@ h3 {
   -ms-user-select: none;
   user-select: none;
 }
-/* content style starts here */
-.content{
```

Comment:
I think this is creating .content selector twice

Reply:
That was what i pointed out to you in the last meeting. I didn't change this file because i am not certain which of the .content selector to be removed. Kindly remove it in your next push to the repo. Thank you

Reply:
On MEET OUR TEAM modal.title you have a typo "Develpers" please correct it